sources/web/datalab/static/datalab.js

```diff
@@ -12,13 +12,17 @@
  * the License.
  */
 
+var debug = {
+  enabled: true,
+  log: function() { console.log.apply(console, arguments); }
+};
+
+function shimWebSockets() {
```

Comment:
add a verb, e.g. shouldShimWebSockets, or canUseWebSockets (returns not equal in this case)